specs/agents/log-onboarding.md

```diff
@@ -0,0 +1,80 @@
+# Log Onboarding
+
+The Agents will be a critical part of log collection onboarding for their
+application logs. This is primarily accomplished via the `log_ecs_formatting`
+configuration option, described below.
+
+In future iterations, the shipping of ECS logs will become more automated by auto-parsing ECS-JSON logs in Filebeat and [automatically shipping log files](https://github.com/elastic/apm/issues/374) that got reformatted via `log_ecs_formatting`.
+
+## `log_ecs_formatting` configuration
+
+Configures the agent to automatically format application logs as ECS-compatible JSON
+(if possible).
+
+|----------------|---|
```

Comment:
Without a table header, it doesn't render properly

```suggestion
|                |   |
|----------------|---|
```

Reply:
Thanks. Thought it was accidentally left-in.